benchmark/index.js

```diff
@@ -421,6 +424,87 @@ test('private box2', (t) => {
   )
 })
 
+test('private box2 group keys', (t) => {
+  const sbot = SecretStack({ appKey: caps.shs })
+    .use(require('../'))
+    .use(require('../encryption-formats/box2'))
+    .call(null, {
+      keys: keys5,
+      path: dirBox2Group,
+    })
+
+  // variables
+
+  // all your groups keys  
+  const groups = 100
+
+  // for messages, the percentage of groups that includes you
+  const percentMe = 30
+
+  const otherKey = Buffer.from(
+    '12345d8f9cbf37f6d7062826f6decac93e308060a8aaaa77e6a4747f40ee1a76',
+    'hex'
+  )
+  sbot.box2.addGroupKey('other', otherKey)
+
+  for (let i = 0; i < groups; ++i)
+  {
+    const startOfKey = String(i).padStart(5, '0')
```

Comment:
OMG, `padStart`, where have you been all my life?

Reply:
context - I like making nicely aligned terminal output and this is such a nice convenience method I didn't know about

Reply:
OMG this is new for me too